python/lsst/daf/butler/core/_butlerUri/http.py

```diff
@@ -345,18 +263,16 @@ def session(self) -> requests.Session:
                 refreshToken(ButlerHttpURI._session)
             return ButlerHttpURI._session
 
-        baseURL = self.scheme + "://" + self.netloc
-
-        if isWebdavEndpoint(baseURL):
-            log.debug("%s looks like a Webdav endpoint.", baseURL)
-            s = getHttpSession()
-        else:
-            raise RuntimeError(f"Only Webdav endpoints are supported; got base URL '{baseURL}'.")
-
+        s = getHttpSession()
         ButlerHttpURI._session = s
```

Comment:
Since we are storing this for all possible remote hosts we wouldn't be able to do the warnings per-host anyhow.